issue_18/RESULTS.md

```diff
@@ -19,11 +19,14 @@ Only `x85_64` results are included at this time.
 | Rocky-8.4-x86_64-dvd1.iso       |   minimal              | media             | VirtualBox VM      | 0                 | 0                   | 0                    | 375             |
 | Rocky-8.4-x86_64-dvd1.iso       |   minimal              | media / efi       | VirtualBox VM      | 0                 | 0                   | 0                    | 412             |
 | | | | |
+| Rocky-8.4-aarch64-boot.iso      | \* graphical-server    | network           | Libvirt VM         | 9                 | 0                   | 0                    | 1239            |
 | Rocky-8.4-x86_64-boot.iso       | \* graphical-server    | network           | VirtualBox VM      | 0                 | 0                   | 0                    | 357             |
```

Comment:
I only see a `minimal-environment` install (rather than graphical-server) in the boot_iso directory for the above Rocky-8.4-x86_64-boot.iso. I think this may be a mistake.

Reply:
I must have pulled these numbers from other results files. I've re-run and will add the missing output files.